win32ss/user/user32/controls/listbox.c

```diff
@@ -2029,6 +2034,8 @@ static LRESULT LISTBOX_HandleMouseWheel(LB_DESCR *descr, SHORT delta )
         pulScrollLines = min((UINT) descr->page_size, pulScrollLines);
         cLineScroll = pulScrollLines * (float)descr->wheel_remain / WHEEL_DELTA;
         descr->wheel_remain -= WHEEL_DELTA * cLineScroll / (int)pulScrollLines;
+        if (cLineScroll < 0)
+            cLineScroll -= descr->page_size;
```

Comment:
@zecarlos1957 : could you confirm to me that's what we want, namely that if cLineScroll is less than zero we make it more negative by subtracting that page_size?

Reply:
MESSAGE("cLine %d -> %d\n",cLineScroll,cLineScroll - descr->page_size);
Scroll to left:
cLine -3 -> -7
cLine -3 -> -7
Scroll to right
cLine 3 -> -1
cLine 3 -> -1
cLine 3 -> -1
This is the output from debug MESSAGE